### PR TITLE
Update contributor profiles

### LINKED ISSUE #122: Edit contributor profile
Need to be able to add sites, change address, etc.

### PR DESCRIPTION
**Pull request by skddc**
* Adds a new page for updating profiles
* Refactors the add-contributor component to allow for updating existing contributors
* Adds input labels to the contributor form and improves placeholders
* Adds event handlers for all contract contributor changes and uses them for updating the UI

closes #122

Caveat: Updating the address is actually not possible with the update-profile contract method, so it will just ignore address changes for now. I think updating both address and profile at once is too risky anyway, so it should be a different form/view/action.

### CODE REVIEW

**Comment by skddc** [comment]
Would be nice if someone could have a quick look at this. Should be very easy to test.

File: app/routes/contributors/edit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    controller.set('attributes', {
+      account: model.account,
+      name: model.name,
+      kind: model.kind,
+      url: model.url,
+      github_username: model.github_username,
+      github_uid: model.github_uid,
+      gitea_username: model.gitea_username,
+      wiki_username: model.wiki_username
```
Comment (bumi) [inline app/routes/contributors/edit.js]: do you think we can use the `accounts` array here already? with adding more services we at some point would want to directly add entries to the accounts array.

Comment (skddc) [reply app/routes/contributors/edit.js]: That should be a separate PR, as it requires quite some changes in the code.

**Comment by galfert** [comment]
Code looks all good to me.

**Pull request merged**